### PR TITLE
Optionally disable reading (GroupValueRead) for sensor and binary_sensor

### PR DESCRIPTION
**Pull request by farmio**
- possiblity to disable reading (GroupValueRead) for sensor and binary_sensor
- move from `address` to `state_address` in HA components sensor and binary_sensor

### CODE REVIEW

File: test/core_tests/config_test.py [hunk 1]
```diff
@@ -376,6 +376,18 @@ def test_config_sensor_percent(self):
                    'Heating.Valve1',
                    group_address_state='2/0/0',
                    value_type='percent',
+                   sync_state=True,
```
Comment (Julius2342) [inline test/core_tests/config_test.py]: Isnt this the default value? (feel free to keep or remove it)

File: xknx/devices/remote_value.py [hunk 2]
```diff
@@ -28,16 +29,22 @@ def __init__(self,
...
+        self.after_update_cb = after_update_cb
         self.payload = None
 
     @property
     def initialized(self):
         """Evaluate if remote value is initialized with group address."""
         return bool(self.group_address_state or self.group_address)
 
+    @property
+    def readable(self):
```
Comment (Julius2342) [inline xknx/devices/remote_value.py]: Hmm, do we find a more explicit name for this property? Not having any better suggestion, but I think this value is not self explaining.

File: test/core_tests/config_test.py (hunk 1 above)
Comment (farmio) [reply test/core_tests/config_test.py]: Yes it is.

File: xknx/devices/remote_value.py (hunk 2 above)
Comment (farmio) [reply xknx/devices/remote_value.py]: I don't have a better name too. 
I think in the context of this module - the direct representation of a KNX Communication object - it's not that bad. I think of it as the pendant to the KNX Read Flag of a GA.

**Pull request merged**